packages/block-library/src/separator/style.scss

```diff
@@ -1,4 +1,6 @@
-.wp-block-separator {
+hr.wp-block-separator {
```

Comment:
Unfortunately, this higher specificity is needed to override the resets. It has some small potential of impacting themes. I'd like some proper testing.